docs/administration-guide/databases/postgresql.md

```diff
@@ -0,0 +1,62 @@
+# Connecting to a PostgreSQL database
+
+In addition to specifying the host, port, database name and user credentials for the database connection, you have the option of securing that connection.
+
+## Use a secure connection (SSL)
+
+### SSL Mode
+
+PostgreSQL databases support different levels of security with their connections, with different levels of overhead.
+
+SSL Mode options include:
+
+- allow
+- prefer
+- require
+- verify-ca
+- verify-full
+
+See the PostgreSQL docs for a table about the different [SSL Modes][ssl-modes], and select the option that works for you.
+
+### SSL root certificate (PEM)
+
+If you set the SSL Mode to either "verify-ca" or "verify-full", you'll need to specify a root certificate (PEM). You have the option of using a **Local file path** or an **Uploaded filed path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```

Comment:
I really don't like the name "Uploaded file path" - I understand what it means after I read the code.
```suggestion
If you set the SSL Mode to either "verify-ca" or "verify-full", you'll need to specify a root certificate (PEM). You have the option of using a **Local file path** or an **Uploaded file path**. If you're on Metabase Cloud, you'll need to select **Uploaded file path** and upload your certificate.
```